docs/api/prompt/index.rst

```diff
@@ -0,0 +1,12 @@
+.. _api_prompt:
+
+=================
+Prompt API
+=================
+Functions in ``FORMATTER_DICT`` for prompt.
+
+.. toctree::
+    :maxdepth: 1
+
+    gitstatus
+    vc_branch
```

Comment:
I thought about that... but those files do not include any functions that does not starts with `_` and have docstrings, should I include them as well?